### PR TITLE
Clarify how to call pana

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Comment by isoos** [comment]
You'll need to modify the README and the tests to make them pass too. (Yeah, it is a bit paranoid)
I can do it later today if you are busy with other stuff..

File: bin/pana.dart [hunk 1]
```diff
@@ -44,8 +44,8 @@ void _printHelp({String errorMessage}) {
     print(red.wrap(errorMessage));
     print('');
   }
-  print('''Usage: pana [<options>] <package> [<version>]
-       pana [<options>] --source path <directory>
+  print('''pana [<options>] --source path <absolute directory path>
```
Comment (sortie) [inline bin/pana.dart]: Can we implement relative paths, e.g. by normalizing them if needed?

Comment (isoos) [reply bin/pana.dart]: Agreed: https://github.com/dart-lang/pana/pull/392

**Comment by isoos** [comment]
Closing PR, https://github.com/dart-lang/pana/pull/392 will handle it better.

**Comment by sortie** [comment]
I think we should still do the --help string improvement, it's a good clarification that it's on pub.dartlang.org by default.

**Pull request closed without merging**